documentation/userguide/docs/shares.md

```diff
@@ -74,21 +85,10 @@ in the request that you are interested in to start working on your request.
 ![add_share](pictures/shares/shares_inbox.png#zoom#shadow)
 
 ## **Add/delete items**
-When you create a share request for a dataset, you still need to add the items (tables or folders) that you want to
-get access to. Initially the share request should be empty of items and in `DRAFT` state, it should look like the following picture.
-
-![add_share](pictures/shares/shares_initial.png#zoom#shadow)
-
-As appears in the picture, by clicking on **Add Item**, the following window will pop up to let you choose a specific table
-or folder in the dataset.
-
-![add_share](pictures/shares/shares_add_window.png#zoom#shadow)
-
-Note that the request is in `DRAFT` status and that
-the items that we add are in `PENDINGAPPROVAL`. They are not shared until the request is submitted and processed.
-
-![add_share](pictures/shares/shares_added.png#zoom#shadow)
-
+If request can be edited (if it is not being processed in the moment), the **Edit** button will appear on top of the page.
```

Comment:
nit: If the request is not being processed, it can be edited by clicking the **Edit** ....

Reply:
Done